sites/public/pages/listings.tsx

```diff
@@ -276,6 +296,17 @@ const ListingsPage = () => {
               options={seniorHousingOptions}
               defaultValue={filterState?.seniorHousing?.toString()}
             />
+            {/* TODO(#515): Add more explanation and an ami percentage
+            calculator to this filter */}
+            <Select
+              id="amiSelect"
+              name={FrontendListingFilterStateKeys.minAmiPercentage}
+              label={t("listingFilters.minAmiPercentageLabel")}
+              register={register}
+              controlClassName="control"
+              options={amiOptions}
+              defaultValue={filterState?.minAmiPercentage?.toString()}
```

Comment:
Is the `toString` necessary here?

Reply:
Yeah, the type of `defaultValue` is `string` not `string | number`